src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-metrics.mdx

```diff
@@ -0,0 +1,1131 @@
+
+For more context on how to use this API, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#attributes-events-insights).
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width={200}>
+        Parameter
+      </th>
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$name`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Required. The desired name for the custom metric.
+      </td>
+    </tr>
+
+        <tr>
+      <td>
+        `$category`
+      </td>
+
+            <td>
+        `string`
+      </td>
+
+      <td>
+        Required. The metric category name, either custom or using a predefined [metric category](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#cat).
+    </td>
+    </tr>
+
+            <tr>
+      <td>
+        `value`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The value of the metric.
+      </td>
+    </tr>
+
+                <tr>
+      <td>
+        `count`
+      </td>
+
+            <td>
+        `int`
+      </td>
+
+      <td>
+        Required. The number of times the metric was recorded.
+      </td>
+    </tr>
+
+                    <tr>
+      <td>
+        `totalValue`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The total value of the metric.
+      </td>
+    </tr>
+
+                    <tr>
+      <td>
+        `exclusiveValue`
+      </td>
+
+            <td>
+        `double`
+      </td>
+
+      <td>
+        Required. The exclusive value of the recording; for example, if the total value contains measurements accounted for elsewhere.
+      </td>
+    </tr>
+
+                        <tr>
+      <td>
+        `countUnit`
+      </td>
+
+            <td>
+        `MetricUnit`
+      </td>
+
+      <td>
+        Required. Unit of measurement for the metric count, including `PERCENT`, `BYTES`, `SECONDS`, `BYTES_PER_SECOND`, or `OPERATIONS`.
+      </td>
+    </tr>
+
+                        <tr>
+      <td>
+        `valueUnit`
+      </td>
+
+            <td>
+        `MetricUnit`
+      </td>
+
+      <td>
+        Required. Unit of measurement for the metric value, including `PERCENT`, `BYTES`, `SECONDS`, `BYTES_PER_SECOND`, or `OPERATIONS`.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+## Examples [#examples]
+
+Here's an example of creating simple custom metrics:
+
+### Java [#java]
+
+```java
+NewRelic.recordMetric("Custom Metric Name","MyCategory", 1.0);
+NewRelic.recordMetric("Login Auth Metric", "Network", 1.0);
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
NewRelic.recordMetric("Custom Metric Name","MyCategory", 1.0)
        NewRelic.recordMetric("Login Auth Metric", "Network", 1.0)

Reply:
```suggestion
```kotlin

    NewRelic.recordMetric("Custom Metric Name","MyCategory", 1.0)
    NewRelic.recordMetric("Login Auth Metric", "Network", 1.0)

```
```